admin/tests/tests.yaml

```diff
@@ -37,6 +37,16 @@ vars:
           tree item will match when searching for the search aliases.
         relevant_for:
           - treeitem
+      - name: searchLabelPattern
+        description: >
+          Template string for the label of tree items in the search results, for example: "{name} ({theme})"
+          Supported parameters:
+          * name: name of the tree item
+          * previous_group: name of the group to which the item belongs (direct parent only)
```

Comment:
IMHO "parent" may be sufficient, parent is a direct ancestor, direct parent is obvious.
previous_group does not really make sense in case of a layer, as this not a group.